linkis-engineconn-plugins/spark/src/main/scala/org/apache/linkis/engineplugin/spark/factory/SparkEngineConnFactory.scala

```diff
@@ -137,6 +137,17 @@ class SparkEngineConnFactory extends MultiExecutorEngineConnFactory with Logging
     sparkConfig.setQueue(LINKIS_QUEUE_NAME.getValue(options))
     sparkConfig.setPyFiles(SPARK_PYTHON_FILES.getValue(options))
 
+    val conf = new util.HashMap[String, String]()
+    addSparkConf(conf, SPARK_DRIVER_HOST.key, SPARK_DRIVER_HOST.getValue(options))
+    addSparkConf(conf, SPARK_DRIVER_PORT.key, SPARK_DRIVER_PORT.getValue(options))
+    addSparkConf(conf, SPARK_DRIVER_BIND_ADDRESS.key, SPARK_DRIVER_BIND_ADDRESS.getValue(options))
+    addSparkConf(
+      conf,
+      SPARK_DRIVER_BLOCK_MANAGER_PORT.key,
+      SPARK_DRIVER_BLOCK_MANAGER_PORT.getValue(options)
+    )
+    sparkConfig.setConf(conf)
```

Comment:
This line needs to delete？

Reply:
This line is necessary because the sparkConf object does not have a directly available method to set parameters such as spark.driver.port. Instead, these parameters need to be assigned values through sparkConf.setConf